src/SWP/Bundle/WebRendererBundle/Controller/DefaultController.php

```diff
@@ -34,8 +36,13 @@ public function indexAction()
             throw $this->createNotFoundException('No homepage configured!');
         }
 
-        return $this->render('index.html.twig', [
+        $tenantContext = $this->get('swp_multi_tenancy.tenant_context');
+
+        $response = $this->render('index.html.twig', [
+            'tenant' => $tenantContext->getTenant(),
```

Comment:
tenant context should be removed from that controller as it has been already removed before in the project - it's not needed, we have twig global to get the tenant from twig templates